src/librustdoc/html/static/css/themes/ayu.css

```diff
@@ -183,11 +183,7 @@ individually rather than as a group) */
 /* FIXME: these rules should be at the bottom of the file but currently must be
 above the `@media (max-width: 700px)` rules due to a bug in the css checker */
 /* see https://github.com/rust-lang/rust/pull/71237#issuecomment-618170143 */
-pre.rust .lifetime {}
-pre.rust .kw {}
 #search-tabs > button:hover, #search-tabs > button.selected {}
```

Comment:
If we decide we should remove the rules, seems like this one should be removed too?